src/elements/element.line.js

```diff
@@ -21,10 +21,14 @@ module.exports = function(Chart) {
 		lineToNextPoint: function(previousPoint, point, nextPoint, skipHandler, previousSkipHandler) {
 			var me = this;
 			var ctx = me._chart.ctx;
+			
+			// This adds an option to all data arrays in the dataset options to let
+			// line bars span gaps. This is useful with sparse data.
+			var spanGaps = this._chart.config.data.datasets[point._datasetIndex].spanGaps
```

Comment:
Agree, and I think it should be done in the [line controller](https://github.com/chartjs/Chart.js/blob/master/src/controllers/controller.line.js#L192), not in the element.